chef-config/lib/chef-config/version.rb

```diff
@@ -20,6 +20,6 @@
 #!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
 module ChefConfig
-  VERSION = '12.5.0.current.0'
+  VERSION = '12.5.0.alpha.0'
```

Comment:
Until we pull `ChefConfig` into its own gem, why not set `VERSION = Chef::VERSION`